src/euphorie/client/browser/country.py

```diff
@@ -382,17 +384,83 @@ def form_action_name(self):
     def template(self):
         return self.index
 
+    @property
+    def author_options(self):
+        sessions_table = self.webhelpers.survey_session_model
+        accounts = (
+            Session.query(Account, func.count(sessions_table.id))
+            .join(sessions_table, sessions_table.account_id == Account.id)
+            .filter(sessions_table.get_context_filter(self.context))
+            .filter(sessions_table.get_account_filter(account=True))
+            .group_by(Account)
+            .order_by(Account.last_name)
+        )
+        return [
+            {
+                "label": account.title,
+                "value": account.id,
+                "selected": "selected"
+                if account.id == self.request.get("author", "")
+                else None,
+            }
+            for account, _ in accounts
+            if account is not self.webhelpers.get_current_account()
+        ]
```

Comment:
I am afraid of the implications of this code against a production DB.
I fear the query might get a bit too expensive and slow.
We had something similar in the todo app and the list of accounts was replaced with an autoselect.
Anyway as you mention this is becoming a sort of alias of the organisation field.
I would ask @pilz to consider of leaving just the organisation field and to hide (at least for the moment this one).

Reply:
Alex said he could live without the author filter, so I'll take it out.

https://github.com/syslabcom/scrum/issues/597#issuecomment-1249183385

Reply:
Come to think of it, there is currently no way that I can see a user's sessions if that user doesn't have an organisation (and has added me to that organisation), so `organisation` and `author` would be equivalent. It would only be beneficial to have the author name in case I know the person's name but not the organisation name, but I think this is highly unlikely.

One more reason to drop the `author` filter.